app/lib/utils.rb

```diff
@@ -4,7 +4,7 @@
 module Utils
   # Converts phone number from e164 to CDC recommended format
   def format_phone_number(phone)
-    cleaned_phone_number = Phonelib.parse(phone).national(false)
+    cleaned_phone_number = phone&.sub('+1', '')
```

Comment:
This is a change that @holmesie and @Bialogs specifically suggested as `Phonelib.parse` is pretty unperformant. In terms of the logic though, all we've been trying to do here with this line of code is to sub out the "+1" from the beginning of each e164 number. In what situation would the functionality be different?

Reply:
`Phonelib.parse` is more flexible in parsing phone numbers, so it will sanitize any strings you pass it to take out parentheses and hyphens and such. Originally I thought `format_phone_number` was used for handling imported phone numbers (in which case this change would be a problem), but it looks like we just call `Phonelib.parse`separately in `import_phone_field`.  Given this looks to only be used in export, it appears to be okay for how we're currently using it.